prow/istio-presubmit.sh

```diff
@@ -64,6 +68,38 @@ else
 fi
 cd $ROOT
 
+# go test setup
+#mkdir -p ~/.kube
+#cp ./.circleci/config ~/.kube/config
+#ln -sf ~/.kube/config ./pilot/platform/kube/config
+
+mkdir -p ~/envoy
+cd ~/envoy
+ISTIO_PROXY_BUCKET=$(sed 's/ = /=/' <<< $( awk '/ISTIO_PROXY_BUCKET =/' $ROOT/WORKSPACE))
+PROXYVERSION=$(sed 's/[^"]*"\([^"]*\)".*/\1/' <<<  $ISTIO_PROXY_BUCKET)
+PROXY=debug-$PROXYVERSION
+wget -qO- https://storage.googleapis.com/istio-build/proxy/envoy-$PROXY.tar.gz | tar xvz
+ln -sf ~/envoy/usr/local/bin/envoy $ROOT/pilot/proxy/envoy/envoy
+cd $ROOT
+
+# go test execution
+time dep ensure -v
+echo FIXME remove mixer tools exclusion after tests can be run without bazel
+time go test $(go list ./mixer/... | grep -v /tools/codegen)
+time go test ./pilot/...
+time go test ./security/...
+time go test ./broker/...
+rm -rf vendor/
+
+if [ "${CI:-}" == 'bootstrap' ]; then
+  # Test harness will checkout code to directory $GOPATH/src/github.com/istio/istio
+  # but we depend on being at path $GOPATH/src/istio.io/istio for imports
+  mv ${GOPATH}/src/istio.io/istio ${GOPATH}/src/github.com
+  ln -sf ${GOPATH}/src/github.com/istio ${GOPATH}/src/istio.io
+  ROOT=${GOPATH}/src/istio.io/istio
+  cd ${GOPATH}/src/istio.io/istio
+fi
+
 # Build
 ${ROOT}/bin/init.sh
```

Comment:
you don't need this since you are doing go tests..

Reply:
and the lines below (bazel test). You essentially don't need bazel at all.

Reply:
Well, I want to do both bazel and go test unless we decide on dropping one of them